pkgs/development/python-modules/pycmarkgfm/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27, cffi, pytest }:
+
+buildPythonPackage rec {
+  pname = "pycmarkgfm";
+  version = "1.0.1";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0wkbbma214f927ikn3cijxsrzkmm5cqz1x4fimrwx9s2wfphj250";
+  };
+
+  propagatedBuildInputs = [ cffi ];
+
+  checkInputs = [ pytest ];
+  checkPhase = ''
+    pytest
```

Comment:
Try to use pytest hook instead:
```nix
  checkInputs = [ pytestCheckHook ];
```

Reply:
Dupe with https://github.com/NixOS/nixpkgs/pull/101672#discussion_r511631728